File: Exercicios/Python/Exercicio10.py
primo = True

num = int(input("Digite um numero: "))

for x in range(2, num):
    if(num % x == 0):
        primo = False
        break

if primo:
    print("O numero " + str(num) + " é primo.")
else:
    print("O numero " + str(num) + " não é primo.")

File: Exercicios/Java/Exercicio10.java
import java.util.Scanner;

public class Exercicio10 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num;
        boolean primo = true;

        System.out.print("Digite um numero: ");
        num = sc.nextInt();
        sc.close();
        
        for(int i = 2; i < num; i++){
            if(num%i==0){
                primo = false;
                break;
            }
        }

        if(primo){
            System.out.println("O numero " + num + " é primo.");
        } else {
            System.out.println("O numero " + num + " não é primo.");
        }
    }
}
